src/config/dynamic-theme-fixes.config

```diff
@@ -14890,6 +14890,15 @@ div.pdf
 
 ================================
 
+manytoon.com
+
+CSS
+.profile-manga {
+   background: var(--darkreader-neutral-background) !important;
```

Comment:
As I presume it needs to be logged in. I can not confirm this fix is needed on the page.

Second thing - if site uses some backround-image then why not use?
`background-image: none !important;`

Reply:
First time I did this I thought about it, but I used this tool wrong and thought it didn't work because of that...